en/services/parameter.md

```diff
@@ -65,6 +66,18 @@ All components must respond to parameter request messages addressed to their ID
 > **Tip** *QGroundControl* by default queries all components of the currently connected system (it sends ID `MAV_COMP_ID_ALL`).
 
 
+## Limitations {#limitations}
```

Comment:
@DonLakeFlyer I've pulled the problem and partial mitigation into a limitations section. The rest of the doc assumes that the parameter set is constant. 
I'm OK that this was the "intent", because otherwise the value setter messages would take an ID.